go/fuzz/pgp_key_read/fuzz.go

```diff
@@ -8,7 +8,7 @@ package libkb
 import "github.com/keybase/client/go/libkb"
 
 func Fuzz(data []byte) int {
-	key, err := libkb.ReadOneKeyFromBytes(data)
+	key, err := libkb.ReadOneKeyFromBytes(nil, data)
```

Comment:
This is a little bit of a pickle.  What if you make `ReadOneKeyFromBytes` return `(key, Warnings, error)`, and then just `Warn` on the warnings when necessary (might also be necessary to conextify the `Warnings` object, it's pretty old...